tasks/lib.yml

```diff
@@ -1,7 +1,7 @@
 ---
 
-- name: install dependent packages
-  yum: name={{item}} state=present
+- name: install dependent packages for red hat derivative distros
+  yum: name={{item}} state=present update_cache=yes
```

Comment:
Updating the cache when installing packages helps with an issue of apt cache being stale for a newly created EC2 instance.